licenses.yaml

```diff
@@ -977,12 +970,22 @@ name: com.squareup.okhttp3 okhttp
 license_category: binary
 module: extensions/druid-kubernetes-extensions
 license_name: Apache License version 2.0
-version: 3.14.9
+version: 4.12.0
 libraries:
   - com.squareup.okhttp3: okhttp
 
 ---
 
+name: com.squareup.okio okio
+license_category: binary
+module: extensions/druid-kubernetes-extensions
+license_name: Apache License version 2.0
+version: 3.2.0
```

Comment:
this version doesn't match the version you specified in in the pom above, so I'm not sure the pom has the intended effect.